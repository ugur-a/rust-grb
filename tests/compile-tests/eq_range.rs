use grb::prelude::*;
mod utils;

fn main() -> grb::Result<()> {
    create_model!(_g, m, x, y, z);
    c!(x + y + 2*z + 1 in 0..=1);
    c!(x + y + 2*z + 1 in ..=1);
    Ok(())
}
