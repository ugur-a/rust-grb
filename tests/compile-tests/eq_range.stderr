error: Use '..' for bounds and range constraints
 --> tests/compile-tests/eq_range.rs:6:28
  |
6 |     c!(x + y + 2*z + 1 in 0..=1);
  |                            ^^^

error: Use '..' for bounds and range constraints
 --> tests/compile-tests/eq_range.rs:7:27
  |
7 |     c!(x + y + 2*z + 1 in ..=1);
  |                           ^^^
